salt/utils/decorators/__init__.py

```diff
@@ -560,6 +560,7 @@ def _decorate(*args, **kwargs):
             return self._call_function(kwargs)
 
         _decorate.__doc__ = self._function.__doc__
+        _decorate.__wrapped__ = self._function
```

Comment:
`__wrapped__` is added on Py3 by `functools.wraps`, want to check for that?

Reply:
Nope, wrapped returns a partial. The `__wrapped__` attribute get's added when the partial is called. This adds a `__wrapped__` attribute to that partial so the test can detect it and behave appropriately.

https://github.com/python/cpython/blob/master/Lib/functools.py#L74

Reply:
:+1: